### PR TITLE
Highly Vital Fix... for grape soda

### LINKED ISSUE #25096: Grape Soda
Maybe i'm just misunderstanding something, but glass of grape soda doesn't look like anything special and it's called glass of what.
default

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
closes #25096

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm [hunk 1]
```diff
@@ -65,6 +65,13 @@
 				icon_state  = "glass_brown"
 				name = "glass of space Cola"
 				desc = "A glass of refreshing Space Cola."
+			if("grapesoda")
+				icon_state = "glass_clear"
```
Comment (Supermichael777) [inline code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: poisonberryjuice is the right color use that instead

Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: reee this is the exact color of the reagent tho

**Comment by Supermichael777** [comment]
actually make 529 glassclear so nothing ends up brown.

File: code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: make a new sprite OR make 529 glassclear so drinks arn't brown

**Comment by ExcessiveUseOfCobblestone** [comment]
yes I agree

**Comment by Fox-McCloud** [comment]
You guys really need to refactor drink icons, holy damn that `if` switch.

**Comment by ExcessiveUseOfCobblestone** [comment]
yeah it could be longer

**Comment by AnturK** [comment]
I'll just close this, the refactor is much better idea.

**Pull request closed without merging**